cfme/tests/containers/test_pods_prop_data_integrity.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.containers.pod import Pod, list_tbl
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+# CMP-9934
+
+
+def test_summary_properties_validation(provider):
+    """
+          This test verifies that the number of running containers in the status summary table
+          is the same number that appears in the Relationships table
+    """
+    sel.force_navigate('containers_pods')
+    ui_pods = [r.name.text for r in list_tbl.rows()]
+    ui_pods_revised = filter(
+        lambda ch: not ch.startswith('metrics'),
+        ui_pods)
+
+    for name in ui_pods_revised:
+        obj = Pod(name, provider)
+        num_container_status_summary = getattr(
+            obj.summary.container_statuses_summary, 'running').text_value
```

Comment:
If you are not parametrizing, it is pointless to use getattr, just do:
```python
num_container_status_summary = obj.summary.container_statuses_summary.running.value
```
And the same for the next line. You see I used .value instead of .text_value, there are numbers right? If yes, then it makes sense to compare numbers, therefore using .value ...